Doc/c-api/type.rst

```diff
@@ -140,6 +140,28 @@ Type Objects
 
    .. versionadded:: 3.9
 
+.. c:function:: void PyType_SetBase(PyTypeObject *type, PyTypeObject *base)
+
+   Set the base type of the giving type. The type stores a strong reference to
```

Comment:
```suggestion
   Set the base type of the given type. The type stores a strong reference to
```

(and a the few other "giving" where I think you should use "given")

Also maybe:
```suggestion
   Set the base type of the giving type. The *type* stores a strong reference to
```

Since you refer to the `type` parameter?